README.md

```diff
@@ -1,135 +1,104 @@
-# Camunda Platform 8 - Get Started
-
-This repository contains a short guide to get started with [Camunda
-Platform 8](https://camunda.com/products/cloud/). It contains instructions on how to
-model your first process, create a user task form, and automate a service task.
-During the guide you will use Console, Modeler, Zeebe, Operate, and Tasklist.
-
-# Table of contents
-
-- [Camunda Platform 8 - Get Started](#camunda-platform-8---get-started)
-- [Table of contents](#table-of-contents)
-- [Repository structure](#repository-structure)
-- [Set up the environment](#set-up-the-environment)
-- [Model the process](#model-the-process)
-  - [Model the user task form](#model-the-user-task-form)
-  - [Configure user task to use form](#configure-user-task-to-use-form)
-  - [Configure service task](#configure-service-task)
-- [Deploy process](#deploy-process)
-  - [Deploy using Camunda Modeler](#deploy-using-camunda-modeler)
-  - [Deploy using Web Modeler](#deploy-using-web-modeler)
-  - [Deploy using zbctl](#deploy-using-zbctl)
-  - [Deploy using code](#deploy-using-code)
-- [Start process instance](#start-process-instance)
-  - [Start instance using Camunda Modeler](#start-instance-using-camunda-modeler)
-  - [Start instance using Web Modeler](#start-instance-using-web-modeler)
-  - [Start instance using zbctl](#start-instance-using-zbctl)
-  - [Start instance using code](#start-instance-using-code)
-- [Complete the user task](#complete-the-user-task)
-- [Complete the service task](#complete-the-service-task)
-- [Further references](#further-references)
-  - [Developer setup of Camunda 8](#developer-setup)
+# Get started with Camunda 8
+
+This repository gets users started with [Camunda 8](https://camunda.com/platform/) and contains instructions on how to:
+
+- Model your first process
+- Create a user task form
+- Automate a service task
+
+During this guide, you will use Console, Modeler, Zeebe, Operate, and Tasklist.
 
 # Repository structure
 
 The repository contains the following folders:
 
-- [_csharp_](csharp/) - C# example for deploy, create instance, and a job worker
-- [_go_](go/) - Go example for deploy, create instance, and job worker
-- [_images_](images/) - Contains screenshots and images for the guide
-- [_java_](java/) - Plain Java example for deploy, create instance, and a job worker
-- [_nodejs_](nodejs/) - Node.js example for deploy, create instance, and a job worker
-- [_process_](process/) - Contains the BPMN process and the user task form
-- [_spring_](spring/) - Spring Boot example for deploy, create instance, and a job worker
+- [_csharp_](csharp/) - C# example for deploy, create instance, and a job worker.
```

Comment:
This syntax was a little odd to me -- do we mean "X example to model, create, and automate your process."?

Reply:
I think so